client/my-sites/plans-features-main/test/plan-type-selector.jsx

```diff
@@ -17,25 +24,36 @@ describe( '<PlanTypeSelector />', () => {
 	};
 
 	test( 'Should show CustomerTypeToggle when kind is set to `customer`', () => {
-		const comp = shallow(
-			<PlanTypeSelector { ...myProps } kind="customer" customerType="personal" />
-		);
+		render( <PlanTypeSelector { ...myProps } kind="customer" customerType="personal" /> );
+
+		expect( screen.getByRole( 'radiogroup' ) ).toHaveClass( 'is-customer-type-toggle' );
+
+		const radios = screen.queryAllByRole( 'radio' );
 
-		expect( comp.find( 'CustomerTypeToggle' ).length ).toBe( 1 );
+		expect( radios[ 0 ] ).toHaveTextContent( 'Blogs and personal sites' );
+		expect( radios[ 1 ] ).toHaveTextContent( 'Business sites and online stores' );
```

Comment:
I'm wondering if there's a better way to test this -- it seems like the radio button index (here and below) is more like an implementation detail. 🤔

Reply:
I don't think the indexes are an implementation detail - they map to what the user sees - the first radio button, the second radio button, and so on, so that's fine IMHO.

Reply:
Agreed, in this case I think order matters.